modules/shared/src/main/scala/org/tessellation/ext/codecs.scala

```diff
@@ -30,4 +34,11 @@ object codecs {
       }
   }
 
+  object NonEmptySetCodec {
+
+    def decoder[A: Decoder: Order: Ordering]: Decoder[NonEmptySet[A]] =
```

Comment:
You can require only `Odering` and create `Order` necessary for `decodeNonEmptySet` using the `Ordering`.